### PR TITLE
Improve the debugger python client messages

### PR DESCRIPTION
**Pull request by polaroi8d**
Changed all the test files. 

JerryScript-DCO-1.0-Signed-off-by: Levente Orban orbanl@inf.u-szeged.hu

### CODE REVIEW

**Comment by zherczeg** [comment]
Patch is good, but #1652 should go in first, which changes a lot of tests.

File: tests/debugger/do_backtrace.expected [hunk 1]
```diff
@@ -1,20 +1,32 @@
...
+(jerry-debugger) Command: s
+Stopped at tests/debugger/do_backtrace.js:18 (in f4() at line:17, col:1)
+(jerry-debugger) Command: backtrace
+Frame 0: tests/debugger/do_backtrace.js:18 (in f4() at line:17, col:1)
 Frame 1: tests/debugger/do_backtrace.js:25 (in foo() at line:21, col:1)
 Frame 2: tests/debugger/do_backtrace.js:33 (in test() at line:30, col:1)
 Frame 3: tests/debugger/do_backtrace.js:40
-(jerry-debugger) Press enter to stop JavaScript execution.
+(jerry-debugger) Command: c
+Press enter to stop JavaScript execution.
```
Comment (zherczeg) [inline tests/debugger/do_backtrace.expected]: These "Press enter to stop JavaScript execution." should be removed in interactive execution.

**Comment by LaszloLango** [comment]
I agree with @zherczeg. The prefix is not needed. Please remove it.

File: jerry-debugger/jerry-client-ws.py [hunk 2]
```diff
@@ -187,7 +190,6 @@ def do_continue(self, args):
         self.exec_command(args, JERRY_DEBUGGER_CONTINUE)
         self.stop = True
         self.cont = True
-        print("Press enter to stop JavaScript execution.")
```
Comment (zherczeg) [inline jerry-debugger/jerry-client-ws.py]: Please keep it in interactive mode. Add

    if not self.non_interactive:

Comment (polaroi8d) [inline jerry-debugger/jerry-client-ws.py]: Done.

**Pull request merged**